src/commonMain/kotlin/exchange.dydx.abacus/state/manager/V4StateManagerAdaptor.kt

```diff
@@ -1390,6 +1394,40 @@ class V4StateManagerAdaptor(
         }
     }
 
+    private fun retrieveLaunchIncentiveSeasons() {
+        val url = configs.launchIncentiveUrl("graphql")
+        if (url != null) {
+            val requestBody = "{\"operationName\":\"TradingSeasons\",\"variables\":{},\"query\":\"query TradingSeasons {tradingSeasons {startTimestamp label __typename }}\"}"
```

Comment:
nit: can are you able remove `__typename` from the raw graphql query? or does that break the query